internal/co_coders/match/on_collab_style_test.go

```diff
@@ -40,34 +40,33 @@ func TestMatchOnCollabStyleSubsetReturnsAMatch(t *testing.T) {
 }
 
 func TestMatchHasAScore(t *testing.T) {
-	aCriteria := user.New([]criteria.CollabStyle{criteria.Team, criteria.Pair})
-	users := []user.User{aCriteria}
-	aMatch := OnCriteria(aCriteria, users)
+	matchCriteria := criteria.Criteria{
+		CollabStyles: []criteria.CollabStyle{criteria.Team}}
+	allMatchUser := user.New(matchCriteria.CollabStyles)
```

Comment:
Why not just pass in matchCriteria to the user constructor?

Reply:
I'll refactor it in the next PR